build/Dockerfile

```diff
@@ -42,4 +41,4 @@ RUN useradd -r -u 1001 -g rhoda-ci-users rhoda-ci-runner
 RUN chgrp -R 1001 $HOME/rhoda-ci && chown -R 1001 $HOME/rhoda-ci && chmod -R 744 $HOME/rhoda-ci
 USER 1001
 
-ENTRYPOINT ["./run_robot_test.sh","-s","-x"]
+ENTRYPOINT ["/bin/sh","-c","./run_robot_test.sh -s -f $RUN_SCRIPT_ARGS -x"]
```

Comment:
What has changed here? You changed the location of ./run_robot_test.sh?
And can you explain what is the difference here between '-x' and '-f'? Is '-f' is by force?